test/tools/EchoArgs/EchoArgs.cs

```diff
@@ -0,0 +1,24 @@
+﻿//---------------------------------------------------------------------
```

Comment:
Can you, please include url from which is was originally obtained to the header?